.gitpod.yml

```diff
@@ -0,0 +1,10 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: pip install -r requirements.txt
+vscode:
+  extensions:
+    - ms-python.python
+
```

Comment:
can you please explain this addition?

Is this related to my request? Because I just asked to update the dependabot.yml. If you check that file, we added all the integrations directory so we need to add this new one too

Reply:
Ok Sir, I've removed it.

Reply:
yeah but still not changed the `dependabot.yml` :P

Reply:
I've updated it sir. Please check.